pyresample/bilinear/xarr.py

```diff
@@ -139,10 +140,16 @@ def _adjust_bands_coordinates_to_match_data(self, data_coords):
         elif 'bands' in self._out_coords:
             del self._out_coords['bands']
 
+    def _get_output_dims(self, data, res):
+        if data.ndim == res.ndim:
+            return data.dims
+        return list(self._out_coords.keys())
```

Comment:
`pyresample/test/test_bilinear.py::TestXarrayBilinear::test_get_sample_from_bil_info` seems to fail without it because the coordinates are in the wrong order (`('x', 'y')` vs. `('y', 'x')`. Based on the currently existing tests, `data.dims` seems to be sufficient and `self._out_coords` are never accessed :thinking:

Reply:
Does that thinking emoji mean you think you need to add more tests?